src/main/grammars/RustParser.bnf

```diff
@@ -523,17 +523,15 @@ EnumVariant ::= OuterAttr* identifier VariantArgs? {
   elementTypeFactory = "org.rust.lang.core.stubs.StubImplementationsKt.factory"
 }
 
-private VariantArgs ::= VariantBlockFields | VariantTupleFields | VariantDiscriminant
+private VariantArgs ::= BlockFields | TupleFields | VariantDiscriminant
 
-BlockFields ::= <<mk_block_fileds FieldDecl>> {
+BlockFields ::= '{' <<field_with_recover FieldDecl>>* '}' {
```

Comment:
But will it be possible to preserve the recovery policy that is defined for `filed_with_recover` if we collapse this rule?

A dozen of tests break when I use

```rust
BlockFields ::= '{' ( !'}' FieldDecl (',' | &'}') )* '}'
```

and hundreds break when I try to make it recoverable :)